jupyter_client/multikernelmanager.py

```diff
@@ -105,9 +108,14 @@ def _context_default(self) -> zmq.Context:
         return zmq.Context()
 
     connection_dir = Unicode("")
+    external_connection_dir = Unicode(None, allow_none=True)
```

Comment:
Since we're silent if this value is set, but not an existing directory, it seems like a validate-handler would be helpful here that, when set, it ensures the value corresponds to an existing directory and presents a helpful message otherwise.

Reply:
On the other side, validating the existence of the directory at start time is kind of restrictive - it doesn't allow for directories that are created after startup. Also, validating the directory only at start time doesn't check if it's removed afterwards.
So I'd prefer to keep the flexibility of the current code.